src/lms-ds-loader/lms_ds_loader/arguments.py

```diff
@@ -0,0 +1,47 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+
+class DbConnection:
+    @staticmethod
+    def build_for_mssql_with_integrated_security(server, port, db_name):

```

Comment:
if DbConnection isn't going to hold any state, consider moving these static methods to a DbConnection module as plain functions.

Reply:
Yeah, the main reason for adding classes would be to encapsulate some data, if you won't, you could use simple functions.

Reply:
Good call. Originally this did encapsulate data, but evolved into just helper methods. Static class of helper methods can be very useful, but not necessary here.

Reply:
Done.